src/Controllers/SettingsController.php

```diff
@@ -222,6 +222,45 @@ public function saveLanguage(Request $request): Response
         return $this->redirect->to('/settings/language');
     }
 
+    public function ifsgCertificate(): Response
+    {
+        $user = $this->auth->user();
+
+        if (!config('ifsg_enabled')) {
+            throw new HttpNotFound('ifsg.disabled');
+        }
+
+        return $this->response->withView(
+            'pages/settings/certificates',
+            [
+                'settings_menu'    => $this->settingsMenu(),
+                'ifsg_certificate' => $user->license->ifsg_certificate,
+                'ifsg_certificate_full' => $user->license->ifsg_certificate_full,
+            ]
+        );
+    }
+
+    public function saveIfsgCertificate(Request $request): Response
+    {
+        $user = $this->auth->user();
+        $data = $this->validate($request, [
+            'ifsg_certificate' => 'optional|checked',
+            'ifsg_certificate_full' => 'optional|checked',
+        ]);
+
+        if (!config('ifsg_enabled')) {
```

Comment:
Eventually that check could be moved up a bit to not run more code than necessary but its fine as it is

Reply:
won't fix